### PR TITLE
New module: support Alibaba Cloud ECS Disk (cloud/alicloud/alicloud_disk) 

### PR DESCRIPTION
**Pull request by xiaozhu36**
SUMMARY

Added a new module alicloud_disk.py that can manage alicloud ecs disk.

COMPONENT NAME

lib/ansible/module_utils/alicloud_ecs.py
lib/ansible/modules/cloud/alicloud/alicloud_disk.py

ANSIBLE VERSION

2.4.0

### CODE REVIEW

File: lib/ansible/module_utils/alicloud_ecs.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
+#
```
Comment (sivel) [inline lib/ansible/module_utils/alicloud_ecs.py:6]: module_utils files should typically carry a BSD license.

Comment (xiaozhu36) [reply lib/ansible/module_utils/alicloud_ecs.py:6]: Hi @sivel ,

Thank you for your feedback. I noticed that other cloud module_utils don't carry a BSD license, and why this module need to do it? 
Looking forward to your feedback, thanks.

Regards,
Guimin

Comment (jctanner) [reply lib/ansible/module_utils/alicloud_ecs.py:6]: @xiaozhu36 if there are other files in module_utils without a bsd license, that is a bug. We will not accept new contributions to module_utils that are not bsd licensed. The explanation can be found here:

https://groups.google.com/forum/#!topic/ansible-devel/zLyEYxRSvaw

Comment (xiaozhu36) [reply lib/ansible/module_utils/alicloud_ecs.py:6]: @jctanner thanks, and I will do it as soon as possible.

**Pull request closed without merging**